src/components/CoinCard/index.tsx

```diff
@@ -0,0 +1,27 @@
+interface IProps {
+  coinInfo: any;
+  price: string;
+}
+
+const CoinCard = ({ coinInfo, price }: IProps) => {
+  return (
+    <div className="bg-gray-700 rounded-md h-20 mx-96 my-16">
```

Comment:
avoid using big values, this will never translate well to mobile, and is not a good practice.

Reply:
avoid giving rigid height. let the content set the height.

Reply:
avoid duplicate divs that do nothing useful.